.github/workflows/CI.yml

```diff
@@ -30,12 +30,21 @@ jobs:
           git config --global user.email "orta+dangersystems@artsy.net"
           git config --global user.name "Danger.Systems"
 
-      - name: Running Tests
+      - name: Running tests
         run: bundle exec rake spec
 
+  run-danger:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.1
+          bundler-cache: true
+
       - name: Running the Dangerfile for this repo
-        if: runner.os != 'Windows'
-        run: |
-          TOKEN='7469b4e94ce21b43e3ab7a'
-          TOKEN+='79960c12a1e067f2ec'
-          DANGER_GITHUB_API_TOKEN=$TOKEN RUNNING_IN_ACTIONS=true echo 'bundle exec danger --verbose'
+        env:
+          DANGER_GITHUB_API_TOKEN: 7469b4e94ce21b43e3ab7a79960c12a1e067f2ec # FIXME Store it in GitHub Actions secrets
```

Comment:
@orta do you think a new GitHub API token can be generated add added as GitHub Actions secret for https://github.com/danger-public?

Reply:
Afraid not, needs to be in the code like above, secrets wont work here because it only runs on branch PRs and not forks

Reply:
Ah, you probably broke the old token by putting it in a single string too, will need to find the password for that account

Reply:
The token is not authorized anymore, if my change deauthorized the token sorry about that.
Indeed storing it as GitHub Actions secrets, will make it not available for fork PRs, but I'm not sure splitting the token in 2 parts, is the best workaround to use in terms of security.

Reply:
Actually, GitHub advises solving the problem differently: https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows.

I'll test that in this PR.

Reply:
In any case the `DANGER_GITHUB_API_TOKEN` must be added as GitHub Actions secrets.

Reply:
Unfortunately using `pull_request_target` or `workflow_run` triggers will not work out of the box. Indeed, the workflow will be executed in the context of the base branch, not of the head branch / PR context.

Reply:
Using env var this way works, but it will not work for forked pull requests:
```
env:
  DANGER_GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```